pom.xml

```diff
@@ -680,5 +680,37 @@
         <id>all-unit-tests</id>
       </profile>
       <!-- TODO: Add a profile to run API tests (integration tests that end in IT.java. See conf/docker-aio/run-test-suite.sh -->
+        <profile>
+            <id>java11</id>
+            <properties>
+                <java.version>11</java.version>
+                <maven.compiler.source>${java.version}</maven.compiler.source>
+                <maven.compiler.target>${java.version}</maven.compiler.target>
```

Comment:
Are "source" and "target" deprecated in Java 9 and later? That's what pull request #6320 was about, I thought.